src/test/scala/sigmastate/helpers/SigmaTestingCommons.scala

```diff
@@ -183,7 +186,6 @@ trait SigmaTestingCommons extends PropSpec
     val randomBytesCount = Gen.chooseNum(1, 20).sample.get
     val randomBytes = Gen.listOfN(randomBytesCount, arbByte.arbitrary).sample.get.toArray
     val bytes = serializer.toBytes(v)
-    serializer.parse(SigmaSerializer.startReader(bytes)) shouldBe v
```

Comment:
let's not remove it, until we prove the generator always yields an empty prefix.

Reply:
Fixed.